native/acter/api.rsh

```diff
@@ -481,6 +491,8 @@ object CalendarEvent {
     fn update_builder() -> Result<CalendarEventUpdateBuilder>;
     /// get RSVP manager
     fn rsvp_manager() -> Future<Result<RsvpManager>>;
+    /// get the reaction manager
+    fn reaction_manager() -> Future<Result<ReactionManager>>;
```

Comment:
we should either go for `reactions` (like on `News`) or `reactions_manager` (like here) and use that consistently. I'd propose to drop `_manager` and just go for `reactions`.

Reply:
Fixed by https://github.com/acterglobal/a3/pull/1208/commits/cdd64defa739398635677d93e9a0efffa3df78ba